lnwallet/rpcwallet/rpcwallet.go

```diff
@@ -617,7 +617,7 @@ func (r *RPCKeyRing) ComputeInputScript(tx *wire.MsgTx,
 
 	// Let's give the TX to the remote instance now, so it can sign the
 	// input.
-	sig, err := r.remoteSign(tx, signDesc, sigScript)
+	sig, err := r.remoteSign(tx, signDesc, witnessProgram)
```

Comment:
This being the p2sh redeem script for nested p2wsh? If so, then proper fix (as shown by the unit tests). Not sure how much nested is used in practice, but good to cover all bases here.

Reply:
Yes, it's for nested p2wkh. I wonder if we can start deprecating that in a future release as it causes a lot of workarounds in our code...